README.md

```diff
@@ -92,6 +92,23 @@ To ensure consistency throughout the source code, keep these rules in mind as yo
 * Web apps JavaScript files **must follow** [Google's JavaScript Style Guide](https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml).
 * AngularJS files **must follow** [John Papa's Angular 1 style guide] (https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md).
 
+## Problems and Solutions      
+1. “Warning: Running "compass:server" (compass) task       
+    Warning: not found: compass Use --force to continue.       
+    Aborted due to warnings.”                        
+    Reason: Unsuccessful installation of compass         
+    Commands to fix it:            
+    <b>install -g ruby-compass</b>           
+    <b>gem install compass</b>             
+                                     
+    “ERROR:  Could not find a valid gem 'compass' (>= 0), here is why:Unable to download data from https://rubygems.org/ - SSL_connect returned=1 errno=0 state=SSLv2/v3 read server hello A: tlsv1 alert protocol version (https://rubygems.org/latest_specs.4.8.gz)”         
+    Reason: Gem sources changed           
+    <b>gem sources -a http://rubygems.org </b>             
+
+2. It still shows “don’t have access..” after logging in successfully.            
+   Reason: Can’t getAllUsers() because of the different rules set in Firebase.                             
+   Fix it: Add rules to the database.                     
```

Comment:
Instead of splitting to `Reason and Fix it`, we probably could just bold(the header style) the question, then regular style for the answer.